main.go

```diff
@@ -23,10 +23,40 @@ const (
 )
 
 var (
-	countFlag  = flag.Int("count", 20, "the number of nodes in the network")
-	targetFlag = flag.Int("target", 10, "the target number of peers")
+	countFlag  = flag.Int("count", 5000, "the number of nodes in the network")
+	targetFlag = flag.Int("target", 70, "the target number of peers")
+	D          = flag.Int("D", 8, "mesh degree for gossipsub topics")
+	Dannounce  = flag.Int("Dannounce", 8, "announcesub degree for gossipsub topics")
+	msgSizeKB  = flag.Int("size", 128, "message size in KB")
```

Comment:
The convention is to suffix the with `-Flag`

Reply:
I'll add the suffixes